### DISCUSSION TITLE
【help】，resolve with dataloader

### DISCUSSION THREAD

**Discussion by Hareis** [feathersjs/feathers#3147] [Q&A]
In version 5, I encountered the same problem as in graphQL. dataloader cannot be used to batch query all data in resolve when using relational query for data associated with an id in a row. When I query 20 or more data, In resolve, do you have a good solution?

**Comment by daffl** [comment]
Dataloader will batch multiple requests at the same time. It can not predict dependent associations. This can however be done manually through [SQL associations](https://feathersjs.com/api/databases/knex.html#associations) or [MongoDB pipelines](https://feathersjs.com/api/databases/mongodb.html#aggregation-pipeline).

**Reply by Hareis** [reply]
So does resolver exist just to be able to do something that doesn't involve network requests
